flink-contrib/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBAsyncSnapshotTest.java

```diff
@@ -96,6 +96,7 @@ public void checkOperatingSystem() {
 	 * the parts that are not asynchronous. If the checkpoint is not done asynchronously the
 	 * test will simply lock forever.
 	 */
+	@Ignore
```

Comment:
If this was working before something seems to be wrong. The `AbstractStreamOperator` is not async right now because of the timers. Maybe it has to do with the order in which state is checkpointed...